awx/api/metadata.py

```diff
@@ -158,7 +158,13 @@ def get_field_info(self, field):
             isinstance(field, JSONField) or
             isinstance(model_field, JSONField) or
             isinstance(field, DRFJSONField) or
-            isinstance(getattr(field, 'model_field', None), JSONField)
+            isinstance(getattr(field, 'model_field', None), JSONField) or
+            (
+                isinstance(field, ManyRelatedField) and
+                field.field_name == 'credentials'
+                # launch-time credentials
+            ) or
+            field.field_name == 'credential_passwords'
```

Comment:
<img width="771" alt="image" src="https://user-images.githubusercontent.com/214912/67808730-9afa8780-fa6d-11e9-981d-65160d5c4a32.png">

Reply:
<img width="1348" alt="image" src="https://user-images.githubusercontent.com/214912/67808895-fd538800-fa6d-11e9-85f0-1785f00c7cbd.png">

Reply:
cc @AlanCoding

Reply:
also works for Vault passwords w/ Vault ID prompts:

<img width="1399" alt="image" src="https://user-images.githubusercontent.com/214912/67809009-40156000-fa6e-11e9-9aeb-e7623ff10cb2.png">